api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/CommonCohortQueries.java

```diff
@@ -784,15 +785,16 @@ public CohortDefinition getMOHPatientsWithVLRequestorResultBetweenClinicalConsul
             + "         p.voided = 0 AND e.voided = 0  "
             + "             AND e.encounter_type = ${6}  "
             + "             AND e.location_id = :location  "
-            + "             AND e.encounter_datetime BETWEEN :startDate AND :endDate  "
+            + "             AND e.encounter_datetime > :startDate AND e.encounter_datetime <= :endDate  "
```

Comment:
So the lower date bound is NOT considered here?

Reply:
Yes, according to the specs that is correct